about-peer-review/pyopensci-related-joss-ropensci.md

```diff
@@ -0,0 +1,47 @@
+# How Are JOSS, rOpenSci and pyOpenSci Related? 
+
```

Comment:
The structure of this page is a bit confusing to me because it does not start out by defining either JOSS or rOpenSci. After reading further I realize that JOSS is defined below, but I don't think rOpenSci gets mentioned again? If it were me I might structure like so:

```
# How Are JOSS, rOpenSci and pyOpenSci Related?
As mentioned on other pages, pyOpenSci works closely with rOpenSci and JOSS. [rOpenSci](link) is a sister organization that directly inspired pyOpenSci; [The Journal of Open Source Scientific Software(JOSS)](link )is a journal and a publisher that provides a mechanism for developers of research software to get credit for their work within academic systems. The JOSS review process is about publication. A review from JOSS will provide you with a citable DOI that is cross-ref enabled.
pyOpenSci aligns closely with the broad mission of JOSS to provide maintainers with credit for their efforts in developing the open source tools necessary to support reproducible open science workflows.

## How is review at pyOpenSci different from the JOSS review process?
We are not a typical publisher or journal. Rather we are a community that provides support for both a diverse group of software maintainers and long term maintenance of our packages. The pyOpenSci review process is different from that of JOSS in a few ways:

1. Our review is specifically design to enforce modern, community-accepted best practices for Python packaging. 
1. pyOpenSci places heavy emphasis on documentation and usability in our reviews and associated standardization of both.
1. pyOpenSci builds community around and visibility for its tools.
1. We will promote packages and package maintainers once they are accepted into our ecosystem.  
1. We support long term maintenance of packages. If the maintainer needs to step down, we will ensure a new maintainer takes over OR sunset and remove the package from our ecosystem.
1. We provide a welcoming forum to ask questions and get help with maintaining your package as needed.  

JOSS reviews are  also [more limited in scope](https://joss.readthedocs.io/en/latest/review_criteria.html) compared to pyOpenSci and the
[submission criteria](https://joss.readthedocs.io/en/latest/review_criteria.html)
are, in places, less stringent than those of pyOpenSci.

## You can improve your package with a review at pyOpenSci and still publish in JOSS
... rest of page here ...
```

Reply:
gosh i've really struggled with this. i decided to make a page about all of the invovled organizations because i thought it was unclear. i think previously i talked about JOSS like 800 times in 800 different places BECAUSE i was worried about people thinking they had to chose between them vs us. Anyway i'll work on this re-org because i think this page is really important and it needs to be clear. i get the joss question ALL OF THE TIME.

Reply:
wow it's SO MUCH BETTER. i was struggling because the JOSS content was literally across several pages and i just needed that fresh set of eyes. pushing here in a few but already i think it's majorly improved with this feedback. THANK YOU!